common/gcloud.py

```diff
@@ -147,3 +146,33 @@ def run_local_instance(startup_script: str = None) -> bool:
     command = ['/bin/bash', startup_script]
     subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
     return new_process.ProcessResult(0, '', False)
+
+
+def create_instance_template(template_name, docker_image, env, project, zone):
+    """Returns a ProcessResult from running the command to create an instance
+    template."""
+    # Creating an instance template cannot be done using the GCE API because
+    # there is no public API for handling some docker related functionality that
+    # we need.
+    command = [
+        'gcloud', 'compute', '--project', project, 'instance-templates',
+        'create-with-container', template_name, '--no-address',
+        '--image-family=cos-stable', '--image-project=cos-cloud',
+        '--region=%s' % zone, '--scopes=cloud-platform',
+        '--machine-type=n1-standard-1', '--boot-disk-size=50GB',
+        '--preemptible', '--container-image', docker_image
+    ]
+    for item in env.items():
+        command.extend(['--container-env', '%s=%s' % item])
+    new_process.execute(command)
```

Comment:
Should you check if this operation succeeded and return None/assert if it failed.

Reply:
I don't think this is needed. This will exception if it fails.